src/System.ServiceModel.Primitives/ref/System.ServiceModel.Primitives.cs

```diff
@@ -4,7 +4,12 @@
 // Changes to this file must follow the http://aka.ms/api-review process.
 // ------------------------------------------------------------------------------
 
-
+namespace System.IdentityModel.Selectors {
+  public abstract partial class X509CertificateValidator {
+    protected X509CertificateValidator() { }
+    public abstract void Validate(System.Security.Cryptography.X509Certificates.X509Certificate2 certificate);
+  }
+}
```

Comment:
There isn't an issue as long as it is a pure subset. You can essentially hide members or interfaces/base types but you cannot introduce new members without having to figure out a plan.

The one question I would have with this type is that is isn't part of ServiceModel. It is part of IdentityModel so I'm not sure if it bellows in ServiceModel.Primitives. How critical is it for the scenario you are trying to accomplish?